pyproject.toml

```diff
@@ -47,6 +47,8 @@ dependencies = [
   "configparser",
   "docopt",
   "eutils>=0.3.2",
+  "importlib_resources",
```

Comment:
Added this dependency to replace `pkg_resources` which gives the following warning...

```
/opt/repos/uta/sbin/ncbi_process_mito.py:9: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  import pkg_resources
```